### DISCUSSION TITLE
sql 更新前的影響行數檢查

### DISCUSSION THREAD

**Discussion by dearsiang** [hhyo/Archery#2452] [Q&A]
Hi 你好

這邊使用Archery 做sql 檢測的時候，這邊執行的sql 語法，是想更新表格其中一欄的數據，理當影響行數應該是一行，但Archery sql 檢測，顯示影響超過606行。

這邊執行後，確實只更新一個欄位數值，只更新一個欄位數值，我猜這應該跟archery sql 沒有什麼關係，只是想問有沒有發生類似的sql 執行經驗分享

**Comment by LeoQuote** [comment]
https://github.com/hanchuanchuan/goInception/issues/605 如有疑问可以前往 goinception 项目提问

**Reply by dearsiang** [reply]
謝謝
